src/test/kotlin/org/rust/ide/refactoring/RsExtractTraitTest.kt

```diff
@@ -0,0 +1,319 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring
+
+import org.intellij.lang.annotations.Language
+import org.rust.RsTestBase
+import org.rust.ide.refactoring.extractTrait.RS_EXTRACT_TRAIT_MEMBER_IS_SELECTED
+import org.rust.lang.core.psi.ext.RsItemElement
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+
+class RsExtractTraitTest : RsTestBase() {
```

Comment:
Let's add some tests with (doc) comments. Currently, the comments are always copied as I see. Later we could add an option to move/copy/ignore it, like Java or Kotlin implementations do with JavaDoc or KDoc. We could mention it in the tracking issue for this refactoring

Reply:
BTW I've opened a tracking issue, feel free to add some further steps there
https://github.com/intellij-rust/intellij-rust/issues/8326